### ISSUE TITLE
A non-closed, but hard state

### ISSUE THREAD

**Issue by bkardell** [WebWeWant/webwewant.fyi#323]
In our conversations during triage we discussed that some wants are what we currently would call "historically unlikely at the moment" or something.  However, the challenge is that numerous things in web history would have been classified as this for many years before we suddenly managed to turn the corner and it became plausible to see a solution.

We'd like to be able to add these as wants but include some commentary that is realistic about its current state and why.

**Comment by aarongustafson** [comment]
Agreed. Based on the setup right now, I think the process would be something like this:

1. Add the Want to the site
2. Kick off the discussion with a thorough review of what (presently) is holding back the want from being resolved

**Comment by aarongustafson** [comment]
In today’s meeting, @tantek called these "moonshots" and we discussed having a section of the site that could house these big ideas and be fodder for researchers and such. We really liked this idea.
